bin/cordlr

```diff
@@ -19,7 +19,6 @@ const flags = minimist(process.argv.slice(2), {
     loader: 'l'
   }
 })
-const plugins = flags._
```

Comment:
This removes usage like:

```js
cordlr cordlr-svg cordlr-color ...
```

Perhaps do something like this:

```js
if (plugins) {
  if (!config.plugins) config.plugins = []
  config.plugins.push(...plugins)
}
```
So  we don't have to deal with 2 objects inside `start`.